g2p_connect_demo/__manifest__.py

```diff
@@ -38,6 +38,8 @@
     "external_dependencies": {"python": ["faker"]},
     "data": [
         "security/ir.model.access.csv",
+        "data/users_data.xml",
```

Comment:
it should be on `demo` param instead of `data`, shouldn't it?

Reply:
This is the demo project.